scripts/test_env.sh

```diff
@@ -7,62 +7,66 @@ TEST=${TEST:-"./test"}
 HOST=${HOST:-"127.0.0.1"}
 IROHA2_CONFIG_PATH="$TEST/peers/config.json"
 IROHA2_GENESIS_PATH="$TEST/peers/genesis.json"
+DEFAULT_N_PEERS=4
```

Comment:
```suggestion
IROHA2_PEER_COUNT=${IROHA2_PEER_COUNT:-"4"}
```

Follow common prefix. While `N` is universally known, the word `DEFAULT` doesn't really convey much. It won't set the number to that value **by default**, but always.